src/Raven.Server/Config/Categories/HttpConfiguration.cs

```diff
@@ -53,16 +53,16 @@ public HttpConfiguration()
         public TimeSetting? KeepAlivePingDelay { get; set; }
 
         [Description("Set Kestrel's HTTP2 max streams per connection. This limits the number of concurrent request streams per HTTP/2 connection. Excess streams will be refused.")]
-        [DefaultValue(null)]
+        [DefaultValue(int.MaxValue)]
```

Comment:
shouldn't be here

Reply:
@ppekrol 

Can you pls explain? 

* Our code sets the default to `int.MaxValue` in:
https://github.com/ravendb/ravendb/blob/9c41a95ea23af627751b15da1d4f3078934ac82d/src/Raven.Server/RavenServer.cs#L198

* I don't see any reference to `null` as being 'no limit' in their documentation about this property:
https://learn.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.server.kestrel.core.http2limits.maxstreamsperconnection?view=aspnetcore-8.0#microsoft-aspnetcore-server-kestrel-core-http2limits-maxstreamsperconnection

Reply:
the default value is `null` which means do not change the defaults, this is a common practice by us. I understand that by accident it matches our behavior, but would like to avoid this change

Reply:
done - reverted to `null`